Documentation/ceph-object.md

```diff
@@ -105,6 +105,46 @@ To directly retrieve the secrets:
 kubectl -n rook-ceph get secret rook-ceph-object-user-my-store-my-user -o yaml | grep AccessKey | awk '{print $2}' | base64 --decode
 kubectl -n rook-ceph get secret rook-ceph-object-user-my-store-my-user -o yaml | grep SecretKey | awk '{print $2}' | base64 --decode
 ```
+## Create an Object Bucket Claim
+
+Creating an Object Bucket Claim (OBC) causes the Rook-Ceph bucket provisioner to create a new bucket or to grant access to an existing bucket. The OBC references a storage class, set up by the administrator, which defines whether or not a new bucket is created, and the retention policy for new buckets. See the [Object Bucket Claim](ceph-object-bucket-claim.md) for more detail on the `CephObjectBucketClaim`.
+
+```yaml
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
```

Comment:
I still don't understand why the first thing shown is a storage class? See my suggested order above?

Reply:
Done